template_debian/distribution.sh

```diff
@@ -358,6 +358,8 @@ EOF
 # ==============================================================================
 function updateLocale() {
     debug "Updating locales"
+    chroot_cmd sh -c 'printf "\\nen_US.UTF-8 UTF-8" >> /etc/locales.gen'
+    chroot_cmd locale-gen
```

Comment:
`locale-gen` is just a fancy wrapper around `localedef`. If adding the above is necessary, then no point in leaving the manual call below.